align_anything/evaluation/benchmarks/mt_bench/vllm_eval.py

```diff
@@ -0,0 +1,342 @@
+    for response in responses:
+        
+        if not response['ref_answer_1']:
+             judge_prompt= prompts[6]['system_prompt']
+             prompt_template = prompts[6]['prompt_template']
+             user_prompt = fill_prompt_template(prompt_template, question_1 = response['question_1'], question_2 = response['question_2'], answer_1 = response['answer_1'], answer_2 = response['answer_2'])
+             system_prompts.append(judge_prompt)
+             user_prompts.append(user_prompt)
+        else: 
+            judge_prompt= prompts[7]['system_prompt']
+            prompt_template = prompts[7]['prompt_template']
+            user_prompt = fill_prompt_template(prompt_template, question_1 = response['question_1'], question_2 = response['question_2'], ref_answer_1=response['ref_answer_1'], ref_answer_2 = response['ref_answer_1'], answer_1 = response['answer_1'], answer_2 = response['answer_2'])
+            system_prompts.append(judge_prompt)
+            user_prompts.append(user_prompt)
+    
+    results = judger.evaluate(system_prompts, user_prompts)
+    for response, system_prompt, user_prompt, result in zip(responses, system_prompts, user_prompts, results):
+            output = result.raw_output.choices[0].message.content
+            score = get_score(output)
+            time = 0
+            while score is None:
+                multi_results=[]
+                multi_results = judger.evaluate(system_prompts=[system_prompt],user_prompts=[user_prompt])
+                output =multi_results[0].raw_output.choices[0].message.content
+                score = get_score(output)
+                time+=1
+                if time >=10:
+                    score = 0
+                    break
+            
+            eval_case.append(
+                 {
+                    'question_id' : response['question_id'],
+                    'system_prompt': system_prompt,
+                    'user_prompt' : user_prompt,
+                    'response': output,
+                    'score': score
+                 }
+             )       
+    
+    return responses, eval_case
+
+def main():
+    parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    _, unparsed_args = parser.parse_known_args()
+    keys = [k[2:] for k in unparsed_args[0::2]]
+    values = list(unparsed_args[1::2])
+    unparsed_args = dict(zip(keys, values))
+    logger = EvalLogger('Evaluation')
+    
+    dict_configs, infer_configs = read_eval_cfgs('mt_bench', 'vLLM')
+
+    try:
+        assert dict_configs or infer_configs, "Config file does not exist or is incomplete."
+    except AssertionError as e:
+        logger.log('error', "Config file is not exist or incomplete.")
+        exit()
+
+    for k, v in unparsed_args.items():
+        dict_configs = update_dict(dict_configs, custom_cfgs_to_dict(k, v))
```

Comment:
```suggestion
        if v == '' or v is None:
            continue
        dict_configs = update_dict(dict_configs, custom_cfgs_to_dict(k, v))
```

Reply:
Thanks for the correction! After that, the program is more robust.